Tests/DependencyInjection/DoctrineExtensionTest.php

```diff
@@ -54,7 +54,51 @@ public function testAutowiringAlias()
 
             $alias = $container->getAlias($id);
             $this->assertEquals($target, (string) $alias, sprintf('The autowiring for `%s` should use `%s`.', $id, $target));
-            $this->assertFalse($alias->isPublic(), sprintf('The autowiring alias for `%s` should be private.', $id, $target));
+            $this->assertFalse($alias->isPublic(), sprintf('The autowiring alias for `%s` should be private.', $id));
+        }
+    }
+
+    public function testEntityManagerAutowiringAlias()
+    {
+        if (! interface_exists(EntityManagerInterface::class)) {
+            self::markTestSkipped('This test requires ORM');
+        }
+
+        if (! method_exists(ContainerBuilder::class, 'registerAliasForArgument')) {
+            self::markTestSkipped('Test requires Symfony 4.2 or higher');
+        }
+
```

Comment:
This is unnecessary as we require Symfony 4.3.3 as lowest supported version.

```suggestion
```

Reply:
Done